app/src/main/res/values/strings.xml

```diff
@@ -68,6 +68,7 @@
 
         This app is just one piece of a bigger series of apps. You can find the rest of them at https://www.simplemobiletools.com
     </string>
+    <string name="swap_prev_next">Swap previous/next keycodes</string>
```

Comment:
add this to the settings section, just below "equalizer". Also add it to every language file, even if obviously not translated.